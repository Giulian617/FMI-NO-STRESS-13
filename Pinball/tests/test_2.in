31 0
23 12 4
19 12 1
25 19 10
18 12 10
26 23 10
16 12 4
8 23 9
17 19 6
22 18 10
9 12 4
31 22 2
2 17 10
4 25 10
30 26 7
29 9 4
20 29 7
3 25 6
5 31 3
24 23 6
21 2 8
7 16 8
6 3 2
27 18 7
14 2 3
1 4 4
15 1 8
10 25 2
11 21 6
13 14 6
28 1 4
