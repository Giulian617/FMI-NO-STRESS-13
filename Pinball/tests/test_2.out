898353287
83333334
730880236
983116890
27777778
541125545
577922082
500000004
730880236
655411260
928847303
250000002
92352093
